### PR TITLE
Add orders queue and associated configuration

### PR DESCRIPTION
**Pull request by ShadowGGlearn**
This PR adds a new Azure Storage queue for processing orders, and updates the website configuration to include the storage account and queue information.

### CODE REVIEW

File: deploy/main.bicep [hunk 1]
```diff
@@ -15,6 +15,7 @@ param storageAccountName string = 'toylaunch${uniqueString(resourceGroup().id)}'
 param environmentType string
 
 var storageAccountSkuName = (environmentType == 'prod') ? 'Standard_GRS' : 'Standard_LRS'
+var processOrderQueueName = 'processorder'
```
Comment (ShadowGGlearn) [inline deploy/main.bicep:18]: Should this be capitalized?

Comment (ShadowGGlearn) [reply deploy/main.bicep:18]: No, storage queues must have lowercase names.

**Review by ShadowGGlearn — COMMENTED** [review]
checked files

**Review by Shadow2k5 — APPROVED** [review]

**Review by Shadow2k5 — APPROVED** [review]
accept

**Review by Shadow2k5 — APPROVED** [review]

File: deploy/main.bicep [hunk 2]
```diff
@@ -26,13 +27,22 @@ resource storageAccount 'Microsoft.Storage/storageAccounts@2019-06-01' = {
   properties: {
     accessTier: 'Hot'
   }
+  resource queueServices 'queueServices' existing = {
```
Comment (Shadow2k5) [inline deploy/main.bicep:30]: q

**Review by Shadow2k5 — APPROVED** [review]

File: deploy/main.bicep (hunk 2 above)
Comment (ShadowGGlearn) [reply deploy/main.bicep:30]: qw

**Pull request merged**